modules/340-extended-monitoring/images/extended-monitoring-exporter/src/extended-monitoring.py

```diff
@@ -301,8 +304,10 @@ def do_GET(self):
             return
 
         if self.path == "/healthz":
-            self.send_response(200)
+            # Fail if metrics were last collected more than 15 minutes ago
+            self.send_response(200 if self.__class__._last_observe > (datetime.now() - timedelta(minutes=15)) else 400)
```

Comment:
```suggestion
            self.send_response(200 if self.__class__._last_observe > (datetime.now() - timedelta(minutes=15)) else 500)
```

That is not a Bad Request, there is something wrong with the server.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses

Reply:
Ok, it makes sense.